<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>matrixDlg</class>
 <widget class="QDialog" name="matrixDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>点石点阵</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QWidget" name="matrixPanel" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>请输入四个汉字：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="contents"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="comitBtn">
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="drawLineBtn">
     <property name="text">
      <string>画线</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>串口设备名称：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="serialDevice"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="downloadBtn">
     <property name="text">
      <string>下载</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="exitBtn">
     <property name="text">
      <string>白白~~</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="informations">
     <property name="text">
      <string>Copyleft @ 2010, &lt;a href=&quot;mailto:tankery.chen@gmail.com&quot;&gt;Tankery Chen&lt;/a&gt; @ &lt;a href=&quot;http://www.dian.org.cn&quot;&gt;Dian Group&lt;/a&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitBtn</sender>
   <signal>clicked()</signal>
   <receiver>matrixDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
